doc/postgresql.md

```diff
@@ -4,7 +4,7 @@
 To seed a Kubernetes/Openshift cluster with an example Postgres resource:
 ```
 $ make cluster/prepare 
-$ make cluster/seed/postgres
+$ make cluster/seed/postgres PROVIDER=openshift
```

Comment:
same as previous comment - Could we add values for PROVIDER in case AWS and GCP?